include/pybind11/detail/internals.h

```diff
@@ -154,49 +154,60 @@ struct type_info {
 
 /// On MSVC, debug and release builds are not ABI-compatible!
 #if defined(_MSC_VER) && defined(_DEBUG)
-#   define PYBIND11_BUILD_TYPE "_debug"
```

Comment:
Very minor & optional: I'd undo the whitespace change here, to reduce the potential for confusion.

Reply:
https://github.com/pybind/pybind11/pull/2602/files?diff=unified&w=1

Reply:
(I can still undo if you think it should be undone - obviously I didn't search out and fix all uneven whitespace ;) )